### PR TITLE
ci: add hello-debian example

### PR DESCRIPTION
**Pull request by cwaldren-ld**
Adds an example that runs in a `debian:bookworm` container.

### CODE REVIEW

File: examples/hello-debian/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# LaunchDarkly Lua server-side SDK Debian Bookworm example
+
+We've built a minimal dockerized example of using the Lua SDK within a Debian Bookworm Docker container. 
+
+## Build instructions
+
+1. On the command line from the **root** of the repo, build the image from this directory with `docker build -t hello-debian -f ./examples/hello-debian/Dockerfile .`.
```
Comment (kinyoklion) [inline examples/hello-debian/README.md:7]: I think this is the wrong command for "this" directory.

Comment (kinyoklion) [reply examples/hello-debian/README.md:7]: I think it needs to be built from the root of the repository.

Comment (cwaldren-ld) [reply examples/hello-debian/README.md:7]: It does:
> 1. On the command line from the **root** of the repo,

Comment (cwaldren-ld) [reply examples/hello-debian/README.md:7]: Oh, maybe "build the image from this directory" is awkward to parse

**Review by kinyoklion — APPROVED** [review]

**Pull request merged**